docs/release-notes/8.1.asciidoc

```diff
@@ -1,10 +1,24 @@
 [[release-notes-header-8.1.0]]
 == 8.1
 
+[discrete]
+[[release-notes-8.1.2]]
+=== 8.1.2
+
+[discrete]
+[[bug-fixes-8.1.2]]
+==== Bug fixes and enhancements
+* Ensures {endpoint-sec} can continue to run on all supported Windows versions by changing the primary signer of the `elastic-endpoint.exe` file from ELASTICSEARCH B.V. to Elasticsearch, Inc. (https://github.com/elastic/endpoint/issues/15[#15]).
```

Comment:
I think it would be helpful to format the signer values in some way, to clearly demarcate where each value starts and ends. I'm not sure if backticks or quotation marks (or something else) would be best, but here's an example using backticks...
```suggestion
* Ensures {endpoint-sec} can continue to run on all supported Windows versions by changing the primary signer of the `elastic-endpoint.exe` file from `ELASTICSEARCH B.V.` to `Elasticsearch, Inc.` (https://github.com/elastic/endpoint/issues/15[#15]).
```